src/14_cal.py

```diff
@@ -21,4 +21,33 @@
 
 import sys
 import calendar
-from datetime import datetime
\ No newline at end of file
+from datetime import datetime
+
+# renderedCalendar = calendar.TextCalendar()
+# date = input("Enter the year, month: ")
+# print(date)
+"""
+if len(sys.argv) == 1:
+elif len(sys.argv) == 2:
+elif len(sys.argv) == 3:
+else:
+print("Invalid command:")
+"""
```

Comment:
Well done here, very simple and clean. Next step is to do what I didn't do either: implement a way to take in user input and parse that and output something into the console. Like the adventure game!